client/packages/requisitions/src/RequestRequisition/api/operations.graphql

```diff
@@ -3,6 +3,7 @@ fragment ItemWithStats on ItemNode {
   name
   code
   unitName
+  defaultPackSize
```

Comment:
only required here due to the data type, though I thought that it might be useful at some point, so left it in